pkg/framework/aries/framework.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package aries
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+	errors "golang.org/x/xerrors"
+)
+
+// Framework provides access to clients being managed by the Framework.
+type Framework struct {
+	opts options
```

Comment:
Why a substructure of options? i.e., Why not have OutboundTransport as part of the Framework struct?

Reply:
I thought it would be cleaner to have config options inside its own struct than at the top level. In case we need something is future which are different from options/config, we can form a new group at top level. Anyways, opts is not exposed and we can refactor as we want. Let me know your thoughts, I can move it if its required.

Reply:
To me, it doesn't seem like "options" once the constructor has completed...

Reply:
[Moved](https://github.com/hyperledger/aries-framework-go/compare/e7224c2166bc28f01e32148341d7f003d8fe83c0..226512799fa5cca78500bfcc4f336200df37bea8#diff-7b2448fdc2181299533bc944239b9744R15).